src/test/java/domain/CrewRepositoryTest.java

```diff
@@ -0,0 +1,24 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class CrewRepositoryTest {
```

Comment:
JUnit5 에서는 테스트 클래스가 public일 필요가 없습니다~!

```suggestion
class CrewRepositoryTest {
```

Reply:
public을 명시하거나 생략하는 이유를 잘 모르겠어서 딱히 신경을 쓰지 않았던 것 같아요.
이번 기회에 찾아보니 public이 필수였던 JUnit 4와 달리 JUnit5에서는 `Less is More` 원칙으로 불필요한 코드의 최소화를 위해 public 키워드 제거를 권장한다고 하네요!

덕분에 새로운 점을 배웠습니다. 피드백 감사합니다. 😄

Reply:
적용했습니다!
[test: 테스트 클래스 public 접근제어자 제거](https://github.com/woowacourse/java-attendance/pull/52/commits/e465aed81991051aca3b7919c2fea04eed4b83ef)